### PR TITLE
feat: email users when they RSVP

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

TODO

- [x] Worry about santizing/encoding - Update: they're just strings that're passed around.  I can't think of a vulnerability
- [x] Add a subscribe link? - Update: it's not done, but this is PR is only a placeholder for the real solution

### CODE REVIEW

**Review by Zeko369 — CHANGES_REQUESTED** [review]
Let's merge this for now, I've figured out how to handle calendar events + updates (https://chat.freecodecamp.org/channel/chapter?msg=isJwqcfWHxLiDsL72) so I'll add at some point before the demo

File: server/controllers/Events/resolver.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { Event, Venue, Chapter, Rsvp } from '../../models';
 import { CreateEventInputs, UpdateEventInputs } from './inputs';
 import { MoreThan } from 'typeorm';
 import { GQLCtx } from 'server/ts/gql';
+import { EmailResolver } from '../Messages/resolver';
+import { format } from 'date-fns';
```
[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply server/controllers/Events/resolver.ts:7]: Thanks, I'll change that.  How about adding https://github.com/benmosher/eslint-plugin-import to catch these?

File: server/controllers/Events/resolver.ts [hunk 2]
```diff
@@ -77,6 +79,32 @@ export class EventResolver {
     });
 
     await rsvp.save();
+    // TODO: sanitize if necessary
+    const inviteURL = new URL('https://www.google.com/calendar/render');
+    const queryObject = {
```
[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply server/controllers/Events/resolver.ts:84]: Oh, neat.  Sure, that looks good.

File: server/controllers/Events/resolver.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zeko369) [reply server/controllers/Events/resolver.ts]: Yes

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply server/controllers/Events/resolver.ts]: Heh, cool.  I'll do that post-demo, since it's not super urgent.

**Comment by allella** [comment]
Just a note that this is part of #110.

**Comment by allella** [comment]
@Zeko369 does this close #110?

**Comment by Zeko369** [comment]
@allella yup

**Pull request merged**